### PR TITLE
fix(tab): spread overrides for tabs too

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: src/tabs/tabs.js [hunk 1]
```diff
@@ -44,6 +50,7 @@ export default class Tabs extends React.Component<TabsPropsT> {
         $orientation: orientation,
         onSelect: () => this.onChange({activeKey: key}),
         children: child.props.title,
+        overrides,
```
Comment (nadiia) [inline src/tabs/tabs.js:53]: wonder if we should merge it with `child.props.overrides` with latest takes precedence

Comment (gergelyke) [reply src/tabs/tabs.js:53]: that's a good point - I have to do that, thanks Nadiia

File: src/tabs/tabs.js [hunk 2]
```diff
@@ -44,6 +50,7 @@ export default class Tabs extends React.Component<TabsPropsT> {
         $orientation: orientation,
         onSelect: () => this.onChange({activeKey: key}),
         children: child.props.title,
+        overrides: child.props.overrides || overrides,
```
Comment (nadiia) [inline src/tabs/tabs.js:53]:
```suggestion
        overrides: mergeOverrides(overrides, child.props.overrides || {})
```

Comment (nadiia) [reply src/tabs/tabs.js:53]: `mergeOverrides` from `import {getOverrides, mergeOverrides} from '../helpers/overrides.js';`

**Pull request merged**